### PR TITLE
Namespace detail

### LINKED ISSUE #45: [Task] Mosaic Page
### Mosaic Page
- showing wrong mosaicID
- showing wrong namespace
- add mosaic active status
- add mosaic end height
- remove transaction table
- remove mosaic `Description` field

### PR DESCRIPTION
**Pull request by AnthonyLaw**
### Added
- Namespace Detail page
- Added mosaicInfo and namespaceInfo DTO

### Update
- update SDK 0.13.3
- refactor Mosaic Detail
- refactor all UI element

close #45 #59

### CODE REVIEW

File: client/src/infrastructure/getNamespace.js [hunk 1]
```diff
@@ -50,24 +50,29 @@ class sdkNamespace {
...
     const namespace = new NamespaceId(name)
     const namespaceService = new NamespaceService(NAMESPACE_HTTP)
-    let namespaceInfo = await namespaceService.namespace(namespace).toPromise()
 
-    let namespaceNames = await NAMESPACE_HTTP.getNamespacesName([namespace]).toPromise()
+    // let namespaceInfo = await namespaceService.namespace(namespace).toPromise()
+    let namespaceInfo = rootNamespace
+
+    // let namespaceNames = await NAMESPACE_HTTP.getNamespacesName([namespace]).toPromise()
+    let namespaceNames = [namespaceName]
```
Comment (evias) [inline client/src/infrastructure/getNamespace.js]: method receives `name` not `namespaceName`

File: client/src/store/namespace.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        'End Height': namespaceInfo.endHeight,
+        'Active': namespaceInfo.active,
+        'Alias': namespaceInfo.alias,
+      }
+      commit('setNamespaceInfo', namespaceInfoObject)
+
+      let namespaceLevels = []
+      namespaceInfo.levels.forEach((el) => {
+        let namespaceLevelObject = {
+          name: "nem",
```
Comment (evias) [inline client/src/store/namespace.js]: why nem here?

Comment (AnthonyLaw) [reply client/src/store/namespace.js]: "nem" should not be here, my mistake 👍

File: client/src/infrastructure/getNamespace.js (hunk 1 above)
Comment (AnthonyLaw) [reply client/src/infrastructure/getNamespace.js]: `namespaceName` is DTO and `import { Endpoint, rootNamespace, subNamespace, namespaceName } from '../config/'`

**Pull request merged**